### PR TITLE
Add translations for "Section" to be used by Nav component

### LINKED ISSUE #4890: Add translations for navigation "Sections" to configs
**Is your feature request related to a problem? Please describe.**
There are currently no translations for a "Sections" word for the navigation menu that should be announced by screen readers. Add these translations to the services configs.

**Describe the solution you'd like**
Translations are in the JIRA ticket 234.

**Describe alternatives you've considered**
N/A

**Testing notes**
Testing will be done as part of the accessibility swarm for navigation (https://github.com/bbc/simorgh/issues/4569).

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves #4890
WSFRONTPGS-234

**Overall change:** Add translations for "Sections" to config

**Code changes:**

- Add translations for "Sections" to 44 service configs.

image
Dark purple = old PAL translations not required in Simorgh

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~~Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)~~
- ~~If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)~~
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by OlgaLyubin — APPROVED** [review]
Looks good to me

File: src/app/lib/config/services/arabic.js [hunk 1]
```diff
@@ -268,6 +268,7 @@ export const service = {
         url: '/arabic/in_depth',
       },
     ],
+    navigationSection: 'أقسام',
```
Comment (AlistairGempf) [inline src/app/lib/config/services/arabic.js:271]: I can't really tell - is this missing something at the end of the word (the left-hand side)? 
image

Comment (OlgaLyubin) [reply src/app/lib/config/services/arabic.js:271]: This is what is on the Jira ticket -  "أقسام" . Seems correct

Comment (j-pendlebury) [reply src/app/lib/config/services/arabic.js:271]: I think that's just a stray quote... 
This is from PAL config:
image
`base.sections = "أقسام"`

**Review by AlistairGempf — APPROVED** [review]
All looks good

**Review by DenisHdz — APPROVED** [review]
Double-checked with the translations in the ticket and they look fine 👍

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**